src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class WorkWithFile {
+    private int supplyInt = 0;
+    private int buyInt = 0;
```

Comment:
lets make this variables local. Our countSupplyAndBuy method could return Array of two numbers